### ISSUE TITLE
Bright White lines on web pages after recent update

### ISSUE THREAD

**Issue by m77e4t** [brave/brave-browser#25304]
## Description 

https://community.brave.com/t/white-horizontal-lines-everywhere/429040

White Lines have started appearing everywhere. Not exactly everywhere, but where there is some sort of section break. Before there were no such white lines. The lines started appearing after the recent major update where Translate feature was introduced in v1.43.88

From brave://flags Dark Mode for all web pages has been turned ON by users

```
Auto Dark Mode for Web Contents

Automatically render all web contents using a dark theme. – Mac, Windows, Linux, ChromeOS, Android, Fuchsia, Lacros

[#enable-force-dark](chrome://flags/#enable-force-dark)
```
Same case for android, by here the option is given directly in settings which controls the flag.

No extension affecting this behaviour (i.e no Dark Reader Extension installed), also extensions are not available on android, whole issue has nothing to do with extensions.

## Actual result:

![a38a0102fbaf6ee1f889a86a5bfc3bbfbdf5f95c_2_225x500](https://user-images.githubusercontent.com/96620334/189261391-2e641bfd-24a8-499c-b178-1c2946819975.png)
![57f1aca5d019778e1b95b3960d359b1921de69c1_2_690x493](https://user-images.githubusercontent.com/96620334/189261435-e19ca1f7-8e2c-4bbb-9642-906eeaa4aa06.png)
![afadcfdd0d7a2680c5efa9a2509634f1c067e7c2_2_690x492](https://user-images.githubusercontent.com/96620334/189261540-5bf22d99-bcd7-4bc2-908a-2c9c3962983c.jpg)
![2095d128a938d47988e92d3824e547dc074e6113_2_690x373](https://user-images.githubusercontent.com/96620334/189261560-047bb184-2268-4884-ba78-6176fe536781.jpg)

https://user-images.githubusercontent.com/96620334/189261680-93b05e7e-cf59-4b03-acec-f1e97e5c4dc2.mp4

## Expected result:
Bright white lines appears on web pages wherever they should be less bright or not at all

## Reproduces how often: 

Easily reproduced after changing the auto dark mode flag on Desktop and Android

## Desktop Brave version:

(Android) Release v1.43.89 (Chromium 105.0.5195.102)
(Linux)  [Version 1.43.89 Chromium: 105.0.5195.102 (Official Build) (64-bit)]

## Android Device details:
   - Install type (ARM, x86):
   - Device type (Phone, Tablet, Phablet): Phone
   - Android version: Android 12

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?  No
- Does the issue resolve itself when disabling Brave Rewards?  NA
- Is the issue reproducible on the latest version of Chrome? No idea

**Comment by m77e4t** [comment]
Brave's twitter is advertising this flag, 'Auto Dark mode for web pages' to users. 
https://nitter.privacy.com.de/brave/status/1568226137901527045#m

It is potentially done at not the right time as such a major bug is due to turning the flag ON.

Do solve this bug, as there will be no problem with turning the flag ON and hence thereafter it can be advertised to users.

**Comment by FatehMammadli** [comment]
![image](https://user-images.githubusercontent.com/76201120/191930118-0a70f26d-15e1-441c-b135-621a41114e4e.png)
![image](https://user-images.githubusercontent.com/76201120/191930121-818c104d-3d7d-4aab-b3f7-5dfb52e944ba.png)
It's fixed for me

**Comment by m77e4t** [comment]
The problem has been solved both on desktop and android automatically after it was solved on the Beta channel. It got solved with the recent update Brave version 1.44.101 Chromium: 106.0.5249.65. It might have solved either from chromium team or brave team, no idea.

Hope it does not come back.
